mongo/integration/unified_spec_test.go

```diff
@@ -55,6 +56,9 @@ var (
 		// i/o timeout.
 		"Ignore network timeout error on find":             godriver2123SkipReason,
 		"Network error on minPoolSize background creation": godriver2123SkipReason,
+		"CreateCollection from encryptedFields.":           godriver2413SkipReason,
```

Comment:
Does this test also try to drop the collection with encrypted fields as an option for `Drop`? Why are we skipping this one?

Reply:
Awesome thanks for adding that local test.